zketcd.go

```diff
@@ -94,14 +94,15 @@ func (z *zkEtcd) mkCreateTxnOp(op *CreateRequest) opBundle {
 		}
 
 		p, respPath = mkPath(op.Path), op.Path
+
+		count := int32(decodeInt64([]byte(s.Get(mkPathCount(pp)))))
+		count++
+		s.Put(mkPathCount(pp), encodeInt64(int64(count)))
 		if op.Flags&FlagSequence != 0 {
-			count := int32(decodeInt64([]byte(s.Get(mkPathCount(pp)))))
 			// force as int32 to get integer overflow as per zk docs
-			cstr := fmt.Sprintf("%010d", count)
+			cstr := fmt.Sprintf("%010d", count-1)
```

Comment:
add a comment on why need to decrease the count here?  (we already increase the counter since blabal.)